src/analog/message.ts

```diff
@@ -144,6 +142,29 @@ export const fromBase64 = ( data: string ) => (
     fromBytes(getBytesFromBase64(data))
 );
 
+export const toMessage = ( commands: Array<Command> ): IMessage => {
+    const commandsBinary = commands.map(command => ({
+        command,
+        ...command.toBinary()
+    }));
+
+    const body = commandsBinary.reduce(
+        (accumulator, {buffer}) => mergeUint8Arrays(accumulator, buffer),
+        new Uint8Array()
+    );
+    const actualLrc = calculateLrc(body);
+
+    return {
+        commands: [], //commandsBinary,
+        lrc: {
+            expected: actualLrc,
+            actual: actualLrc
+        },
+        buffer: mergeUint8Arrays(body, new Uint8Array([actualLrc])),
```

Comment:
how about simpler approach?

new Uint8Array([...body, actualLrc])

here in other places

Reply:
fixed